client/package.json

```diff
@@ -11,11 +11,13 @@
   "dependencies": {
     "axios": "^0.21.0",
     "core-js": "^3.6.5",
+    "debug": "^4.2.0",
+    "package.json": "^2.0.1",
```

Comment:
This package says that it is deprecated when installing and to use a different one:

![Error](https://i.imgur.com/zjNVOCo.png)

Reply:
Honestly not sure how/when this made it in.